bot/bot_discord.py

```
@@ -13,32 +14,81 @@ class BotOraculo(commands.Bot):
-        # Registra os comandos de aplicação
-        await self.tree.sync()
-        print(f'{self.user} conectado!')
+    async def on_ready(self) -> None:
+        """Callback chamado quando o bot está pronto."""
+        try:
+            # Registra os comandos de aplicação
+            await self.tree.sync()
+            self.logger.info(f'Bot {self.user} conectado e pronto!')
+            self.logger.info(f'Conectado a {len(self.guilds)} servidores')
+        except Exception as e:
+            self.logger.error(f"Erro durante inicialização: {str(e)}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)